main.rb

```diff
@@ -0,0 +1,53 @@
+require "/Users/kirillgavris/Desktop/Library/autoload"
+
+def build_author
+    name = FFaker::Book.author
+    biography = FFaker::Book.description
+    Author.new(name, biography)
+  end
+
+  def build_book(author)
+    title = FFaker::Book.title
+    Book.new(title, author.name)
+  end
+
+  def build_reader
+    name = FFaker::Name.name
+    email = FFaker::Internet.email
+    city = FFaker::Address.city
+    street = FFaker::Address.street_name
+    house = rand(1 - 10_000)
+    Reader.new(name, email, city, street, house)
+  end
+
+  def build_order(book, reader)
+    date = Time.now.utc.strftime('%d.%m.%Y')
+    Order.new(book.title, reader.name, date)
+  end
+
+  def fill_author
+    3.times do
+      author = build_author
+      @authors.push(author)
+    end
+  end
+
+  def fill_book
+    6.times do
+      book = build_book(@authors.sample)
+      @books.push(book)
+    end
+  end
+
+  def fill_reader
+    10.times do
+      reader = build_reader
+      @readers.push(reader)
+    end
+  end
+
+  def fill_order
+    16.times do
+      @orders.push(build_order(@books.sample, @readers.sample))
+    end
+  end
```

Comment:
1. Move all those code into some extension. main.rb is needed to show the results of your task, for example:

```
lib = Library.new
lib.create_data
# lib.load_storage (commented so the reviewer could see and check how the load works)
lib.save_to_storage
lib.top_reader
lib.top_book
```

Your service with data generation should look like this

```
class FakeDataGenerator
  class << self
    def call
      generate_data

      {
        authors: @authors,
        books: @books,
        readers: @readers,
        orders: @orders
      }
    end

    private 
...all methods for data generation here
```
  class << self would let you do FakeDataGenerator.call, without initializing an instance

Reply:
moved and added class and private method